src/data/nat/choose.lean

```diff
@@ -100,8 +101,142 @@ theorem add_pow [comm_semiring α] (x y : α) (n : ℕ) :
 (commute.all x y).add_pow n
 
 /-- The sum of entries in a row of Pascal's triangle -/
-theorem sum_range_choose (n : ℕ) : 
+theorem sum_range_choose (n : ℕ) :
   ∑ m in range (n + 1), choose n m = 2 ^ n :=
 by simpa using (add_pow 1 1 n).symm
 
+/-
+Facts about more specific binomial coefficients and their sums
+-/
+
+private lemma succ_not_le {m : nat} (t : m.succ ≤ m) : false :=
```

Comment:
This is `nat.not_succ_le_self`.

Reply:
What should I have done to make `library_search` find this? It's verbatim a theorem in the standard library, but `library_search` failed to implement its body.

Reply:
@semorrison :arrow_up: